src/file.ts

```diff
@@ -251,6 +251,7 @@ export interface MoveCallback {
 
 export interface MoveOptions {
   userProject?: string;
+  preconditionOpts?: PreconditionOptions;
```

Comment:
Where is `PreconditionOptions` defined?

Reply:
It was part of the earlier PR for gcs-resumable-upload: https://github.com/googleapis/nodejs-storage/blob/0c1fa3934a52a608366a8c6c798c43516dd03dbf/src/storage.ts#L61